go.mod

```diff
@@ -0,0 +1,11 @@
+module ratls
```

Comment:
I would opt for `gramine-ratls`,  the package name and the file name should also be named `gramine-ratls`

Reply:
Golang doesn't allow `-` in the package name of the source file, will rename it to `gramine_ratls`. Does this work?